gensim/test/test_hdpmodel.py

```diff
@@ -51,12 +52,29 @@ class TestHdpModel(unittest.TestCase, basetests.TestBaseTopicModel):
     def setUp(self):
         self.corpus = mmcorpus.MmCorpus(datapath('testcorpus.mm'))
         self.class_ = hdpmodel.HdpModel
-        self.model = self.class_(corpus, id2word=dictionary)
+        self.model = self.class_(corpus, id2word=dictionary, random_state=np.random.seed(0))
 
     def testShowTopic(self):
```

Comment:
It's already in https://github.com/RaRe-Technologies/gensim/blob/9a3329cfccfcf98e9b88f5f52437f1049f4e3c44/gensim/test/basetests.py#L28